### DISCUSSION TITLE
CDN's and FQDN's that are being used in erase-install

### DISCUSSION THREAD

**Discussion by shan1784** [grahampugh/erase-install#461] [General]
Hello, I posed a question yesterday about erase-install failing at the "Sending password" step when attempting to upgrade to 14.1. I talked with our security and they asked for specific CDN's and FQDN's being called in this script. Could someone shed some light on what these might be?

**Comment by grahampugh** [comment]
Send them this: https://support.apple.com/en-us/HT210060

**Reply by shan1784** [reply]
Thank you for this. What sections in particular would involve erase-install? Sorry for extra questions but I know they will ask me.

**Reply by grahampugh** [reply]
Look for anything to do with software updates I guess. Sorry but I cannot be more specific, and your security team really should be allowlisting everything on that page if you don't want further problems down the line.

**Reply by shan1784** [reply]
will do thank you much!!!
